regression/esbmc/github_2338_2/main.c

```diff
@@ -0,0 +1,9 @@
+#include <assert.h>
+
+int main() {
+  unsigned int a = 0xFFFFFFFF;
+  unsigned int result = -a;
+  assert(result == 1);
```

Comment:
I think this assert is never actually triggered, because the negation will cause an error beforehand.

Reply:
Assertion removed.